src/Microsoft.Diagnostics.Runtime.Utilities/Microsoft.Diagnostics.Runtime.Utilities.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Diagnostics.Runtime" Version="2.3.405304" />
+    <PackageReference Include="Microsoft.Diagnostics.Runtime" Version="4.0.0-beta.24320.3" />
```

Comment:
Why don't you use a ProjectReference here?

Reply:
It's a separate library with a separate nuget package and dependencies.  It also (theoretically) can version and move independently of the original library, though they've been roughly in sync in the past.  This lets me put experimental/useful tooling there without polluting the main ClrMD library.  These being loosely coupled has been a useful property.